Gemfile

```diff
@@ -4,6 +4,7 @@ source 'https://rubygems.org'
 gemspec
 gem "rubysl", platform: :rbx
 gem "json", platform: :rbx
+gem "connection_pool"
```

Comment:
This should probably be a dev dependency, since it's only being used by tests and we wouldn't want all consumers to depend on it.

Reply:
Right - I'm not sure I fully get the difference between adding a dependency to `Gemfile` and adding it as a dev dependency. Perhaps adding as a dev dependency when we actually want consumers to install it (when running in dev mode)? Dependencies in the Gemfile won't be marked as dependencies for verdict.

Reply:
Hmm, this was somewhat confusing. I think we want to keep the development dependencies in https://github.com/Shopify/verdict/blob/1b65cf6a0f966814fc56de9ebd7a5b8643c7de2d/verdict.gemspec#L22 when possible.

Reply:
Ok, fixed.